action.yml

```diff
@@ -1,9 +1,9 @@
 name: 'Elm Validate'
 description: 'Runs Elm through elm-format and elm-review then compiles it'
 compile-working-directory:
-    description: 'Working directory of the compile step'
+    description: 'Working directory of linter/review/compile step'
     required: false
-    default: 'src'
+    default: '.'
```

Comment:
Just to be registered in the PR, looks like this default does not makes difference at all. Changing it did not change any behavior, as seen in https://github.com/PaackEng/wms-web/actions/runs/537721028